byzcoin/statetrie.go

```diff
@@ -90,6 +95,14 @@ func (t *stagingStateTrie) GetIndex() int {
 	return int(index)
 }
 
+func (t *stagingStateTrie) StoreAllToReplica(scs StateChanges) (ReadOnlyStateTrie, error) {
```

Comment:
Can you add a comment on this one? Like" it copies the stagingStateTrie but keeps the disk-backed tries".

Also I know its not common but wrapped errors are much easier to debug. Can you format the error with something like `"replica failed to store state changes: " + err.Error()`?